app/models/soda.rb

```diff
@@ -0,0 +1,18 @@
+class Soda < ApplicationRecord
+  belongs_to :creator, class_name: 'User'
+  belongs_to :last_editor, class_name: 'User'
+  belongs_to :country
+  has_many :photos
+  has_and_belongs_to_many :flavors
```

Comment:
I felt the same about hierarchy. I don't think we necessarily need it.

Compound flavor: question, for cherry root beer, would it be root beer or the most detailed description which is cherry root beer? I am trying to understand the concept.

_"I was thinking may be we can create in the notes field: add flavors to soda." - can you clarify what this means?_  I was thinking about the option to be able to add more flavors when the person tries the soda. For example I try a soda flavored "London Fog" and I have no idea what actually it is. Then when I write a review/notes on this soda, I am adding flavors like cream, butter, sugar or smth like this. Just a thought. Not necessarily what we have to account for.